app/code/Magento/Shipping/Model/Tracking/Result/Status.php

```diff
@@ -1,21 +1,29 @@
 <?php
+declare(strict_types=1);
+
 /**
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
 namespace Magento\Shipping\Model\Tracking\Result;
 
 /**
- * Fields:
- * - carrier: carrier code
- * - carrierTitle: carrier title
+ * Tracking Status DataObject
+ *
+ * @method string getCarrier()
```

Comment:
all getters could return `null` in case when model isn't filled with data.
```suggestion
 * @method string|null getCarrier()
```